tests/reboot_cause_test.py

```diff
@@ -68,8 +73,84 @@ def test_reboot_cause_history(self):
         print(result.output)
         assert result.output == expected_output
 
+    # Test 'show reboot-cause history all'
+    def test_reboot_cause_history_all(self):
+        runner = CliRunner()
+        result = runner.invoke(show.cli.commands["reboot-cause"].commands["history"], ["all"])
+        print(result.output)
```

Comment:
No assertions present, what is being tested here? Please add assertions to compare the expected output and the output obtained from invoking the command even for positive cases

Reply:
@gpunathilell Done